docs/zh-cn/develop/Commands.md

```diff
@@ -0,0 +1,58 @@
+# 命令
+
+大多情况下，你应该使用 `/` 类型的命令，但有时，我们会需要一些额外的发挥空间。
+
+## 创建命令
+
+创建新的命令很简单，你要做的是：实现 `Command` 接口，通过创建一个[编码器](https://forge.gemwire.uk/wiki/Codecs)来序列化或反序列化命令，然后用 `CommandTypes.register()` 来注册编码器。
+
+让我们创建一个能够将字符串转化为标准输出的命令。
+
+```java
+public record DummyCommand(String text) implements Command {
+
+    @Override
+    public boolean execute(EventContext context) {
+        System.out.println(text());
+        return true; //执行成功则返回
+    }
+
+    @Override
+    public CommandType type() {
+        return null;
+    }
+}
+```
+
+接下来，我们再为这个命令[编码器](https://forge.gemwire.uk/wiki/Codecs)创建一个编码器。
```

Comment:
I wonder if there's a translated resource about Codecs.

Reply:
Unfortunately, I didn't find any translated resource about this.